common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/RedundantTokenRemover.scala

```diff
@@ -0,0 +1,41 @@
+package com.gu.mediaservice.lib.cleanup
+import com.gu.mediaservice.model.ImageMetadata
+
+object RedundantTokenRemover extends MetadataCleaner {
+  val toRemove = List(
+    "Handout",
+    "HANDOUT",
+    "HO",
+    "HO HANDOUT",
+    "Pool",
+    "POOL",
+    "POOL New",
+    "STRINGER",
+    "stringer",
+    "Stringer",
+    "STR",
+    "-",
+    "UNCREDITED",
+    "Uncredited",
+    "uncredited",
+    "XXSTRINGERXX xxxxx"
+  )
+
+  override def clean(metadata: ImageMetadata): ImageMetadata = metadata.copy(
+    byline = metadata.byline.map(removeHandoutTokens).filter(_.trim.nonEmpty).map(_.trim),
+    credit = metadata.credit.map(removeHandoutTokens).flatMap { c =>
+      if (c.isEmpty) {
+        metadata.credit.flatMap(c => c.split("/").lastOption)
+      } else {
+        Some(c)
+      }
+    }.map(_.trim),
+  )
+
+  def removeHandoutTokens(text: String): String = {
+    text.split("/").filter { tok =>
+      val trimmedToken = tok.trim
+      !toRemove.contains(trimmedToken)
```

Comment:
nit: should we `!toRemove.contains(trimmedToken.toLower())` and then the `toRemove` val can be smaller?